packages/@okta/i18n/src/properties/login_ok_PL.properties

```diff
@@ -850,6 +850,7 @@ oie.yubikey.description = 》Îñšéŕţ ţĥé ÝûƀîĶéý îñţö å ÛŠ
 oie.yubikey.passcode.label = 》Îñšéŕţ ţĥéñ ţåþ ýöûŕ ÝûƀîĶéý ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.custom.app.authenticator.title = 》Ĝéţ å þûšĥ ñöţîƒîçåţîöñ 한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
 oie.custom.app.authenticator.description = 》{0} îš åñ åûţĥéñţîçåţöŕ åþþ، îñšţåļļéð öñ ýöûŕ þĥöñé، ûšéð ţö þŕöṽé ýöûŕ îðéñţîţý 홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
+oie.smartcard.authenticator.description = 》Ûšé å þĥýšîçåļ šɱåŕţ çåŕð، šûçĥ åš ÞÎṼ öŕ ÇÅÇ، ţö šîĝñ îñ ヸ€홝한Ӝฐโ⾼ئ䀕ヸ€홝한Ӝฐโ《
```

Comment:
Not sure if this was intentional, but don't translations get generated after the PR is merged?

Reply:
Yes it is intentional. We need to run [pseudo-loc](https://okta.slack.com/archives/CAZH2MWEL/p1652720088043439?thread_ts=1652719963.457889&cid=CAZH2MWEL) to generate this file else [linting](https://okta.slack.com/archives/CAZH2MWEL/p1652720487091099?thread_ts=1652719963.457889&cid=CAZH2MWEL) will fail.